### DISCUSSION TITLE
Beam Element

### DISCUSSION THREAD

**Discussion by ywan459** [kinnala/scikit-fem#1044] [Q&A] [answered]
Hi, 

I have accidentally found the class called skfem.mesh.MeshLine1. Does this mean the potential support of the beam element by scikit-fem? Thanks

Regards
ywan459

**Comment by gdmcbain** [comment]
Yes, see the [Euler&ndash;Bernoulli beam](https://scikit-fem.readthedocs.io/en/latest/listofexamples.html#example-34-euler-bernoulli-beam) example.  The element is ([L15](https://github.com/kinnala/scikit-fem/blob/6159aa309cd182578405ee8a84553d9a6d7462de/docs/examples/ex34.py#L15))

```python
e = ElementLineHermite()
```

**Reply by ywan459** [reply]
Thanks! 

Following this: https://en.wikipedia.org/wiki/Euler%E2%80%93Bernoulli_beam_theory,
I guess in this example, the product of the elastic modulus (E) and the second moment of the cross-sectional area (I) was 1?

Regards
ywan459

**Reply by gdmcbain** [reply, marked as answer]
Yes, also for unit length and force per unit length, or rather (following Wikipedia's nomenclature) the coordinate _x_ has been normalized by the length _&ell;_ and the displacement by _q&ell;&sup2;/EI_, where _q_ is the force per unit length.

**Reply by ywan459** [reply]
True, it is very clear now. Thanks for sharing and explanation. This is the full answer. 

Regards
ywan459
